app/views/game.html.erb

```diff
@@ -0,0 +1,17 @@
+<form action='/api/guess' class='playground-form'>
+  <div class="codes">
+    <% @game&.code_length&.times do |index| %>
```

Comment:
So, if you have no game, it will show empty block, maybe it should be handled in other way? 
Something like, show a n error message instead or diferent form?